src/views/task_list_view.js

```diff
@@ -0,0 +1,69 @@
+import Backbone from 'backbone';
+import _ from 'underscore';
+import $ from 'jquery';
+import TaskView from '../views/task_view';
+import Task from '../models/task'
+
+const TaskListView = Backbone.View.extend({
+  initialize: function(params) {
+    this.template = params.template;
+    this.listenTo(this.model, 'update', this.render);
+  },
+  render: function() {
+    this.$('#todo-items').empty();
+    this.model.each((task) => {
+      const taskView = new TaskView({
+        model: task,
+        template: this.template,
+        tagName: 'li',
+        className: 'task',
+      });
+      this.listenTo(taskView, 'editMe', this.editTask);
```

Comment:
Should editing individual tasks be done in the `TaskView` instead of `TaskListView` instead?

Then again your way demonstrates triggering and alerting the larger view.

Reply:
Yeah, my understanding of the lecture as it had been was to demonstrate that editing should change the larger view's form, so the larger view needed a reference to it. It's not necessarily the best kind of "editing function" though (a real app would probably rather have just a form inside of the task view), do you think I should change it?